### PR TITLE
add returning promise example for `returns` tag

### PR DESCRIPTION
**Pull request by zhengzongyi**

### CODE REVIEW

File: content/en/tags-returns.md [hunk 1]
```diff
@@ -73,3 +73,20 @@ function sum(a, b, retArr) {
...
 {% endexample %}
+
+{% example "Returns a promise" %}
+
+```js
+/**
+ * Returns the sum of a and b
+ * @param {Number} a
+ * @param {Number} b
+ * @returns {Promise<Number>} Promise object represents the sum of a and b
```
Comment (hegemonic) [inline content/en/tags-returns.md]: nit: change `Number` to `number` in all three type expressions

File: content/en/tags-returns.md [hunk 2]
```diff
@@ -73,3 +73,20 @@ function sum(a, b, retArr) {
...
+ */
+function sumAsync(a, b) {
+    return new Promise(function(resolve, reject){
```
Comment (hegemonic) [inline content/en/tags-returns.md]: nit: space before `{`

File: content/en/tags-returns.md (hunk 1 above)
Comment (zhengzongyi) [inline content/en/tags-returns.md]: @hegemonic  Done. Thank you for the suggestions.

**Comment by hegemonic** [comment]
Thanks for making the changes!

**Pull request merged**